spec/datadog/appsec/actions_handler/serializable_backtrace_spec.rb

```diff
@@ -0,0 +1,176 @@
+# frozen_string_literal: true
+
+require 'datadog/appsec/spec_helper'
+
+RSpec.describe Datadog::AppSec::ActionsHandler::SerializableBacktrace do
+  let(:location_struct) do
+    Struct.new(:path, :lineno, :label) do
```

Comment:
Maybe we can use `Thread::Backtrace::Location` for a good reference?

Reply:
done using `instance_double` in [bf5323c](https://github.com/DataDog/dd-trace-rb/pull/4526/commits/bf5323c792643880171a19cd121f8eba79b10617)